### PR TITLE
Faster ci

### PR DESCRIPTION
**Pull request by djhi**

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -22,8 +22,23 @@ jobs:
...
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Use Node.js LTS
+      uses: actions/setup-node@v1
+      with:
+        node-version: '14.x'
+    - uses: bahmutov/npm-install@v1
+      with:
+        install-command: yarn --frozen-lockfile --non-interactive --silent
     - name: Lint
```
Comment (fzaninotto) [inline .github/workflows/test.yml:42]: shouldn't we put that in the typecheck phase, too?

Comment (djhi) [reply .github/workflows/test.yml:42]: It's more about having all jobs taking approximately the same time

Comment (djhi) [reply .github/workflows/test.yml:42]: unit-test is already the fastest job so it make sense to keep the linting there

**Comment by fzaninotto** [comment]
Awesome!

**Pull request merged**